Frontend/src/data/rental/inputs.js

```diff
@@ -234,6 +257,10 @@ export default {
       component: AutocompleteInput,
       nobind: true,
       props: {
+        localSorting: true,
+        sortByMatchedKeywords: true,
+        itemSortFunction: () => sortItemByIdOrName,
+        localFiltering: true,
```

Comment:
Why do we need localFiltering? We already filter in the searchFunction, so I think this might not be necessary?

Reply:
Took me quite a bit of devbugging to find out which set of flags made the sorting work. 

Can be seen here https://github.com/pstanoev/simple-svelte-autocomplete/blob/fc46ab3bb4dc908ff5cc8773628127a68dde7c2b/src/SimpleAutocomplete.svelte#L504

if localfiltering is False it will not trigger the sorting. Might be mistaken though, didnt look too close

Reply:
Ah, I see, you are right.